### DISCUSSION TITLE
Contract reward multiples over quotas

### DISCUSSION THREAD

**Discussion by Malcolm-Q** [Malcolm-Q/LC-LateGameUpgrades#218] [Ideas]
ImpulsiveLad:

The contract reward should increased on quotas.

The cost of doing it and wagt you get out for completing it can be chosen but regardless of how high it is, the later the quota you get to, the worse of an idea it would be to go for it, if you have a 1500 quota and your contract is on one of the first 3 planets it’s financially better to just go to titan or another paid planet .

**Comment by ImpulsiveLass** [comment]
damn I just noticed the typo, sent that from bed at like 3am

**Comment by WhiteSpike** [comment]
Closing due to 3.3.0 adding configuration for the contract rewards be influenced by the current profit quota by given percentage.
